docs/manage/remotes.md

```diff
@@ -0,0 +1,92 @@
+
+When you configure a remote or a sub-part, the main robot part can access all the components and services configured on the remote robot part as though they were entities of the main robot part.
+This is a one-way connection: The main robot part can access the resources of the remote robot part, but the remote robot cannot access the resources of the robot part remoting into it.
+
+<img src="../img/remotes/remotes-diagram.png" alt="Example of a remote and a two part robot where the main (and only) part of robot 1 remotes into the main part of robot 2, and thus has access to all resources of robot 2." >
+
+## Configuration
+
+### Configure a sub-part
+
+You can make a multi-part robot by first configuring one part which is the "main" part, and then configuring one or more sub-parts.
+The main part will be able to access the resources of its sub-parts.
+Sub-parts will *not* have access to the resources of the main part.
+
+Use the parts drop-down menu in the top banner of your robot's page on the [Viam app](https://app.viam.com) to add a new sub-part:
+
+![The Viam app interface with the part drop-down open. A new part called "my-sub-part" is being created.](../img/remotes/sub-part-config.png)
+
+<br>
+
+{{% alert title="Note" color="note" %}}
+
+When you create a sub-part, a connection is established between the main part and the sub-part without you needing to explicitly configure a remote.
+
+{{% /alert %}}
+
+### Configure a remote
+
+To establish a connection between a part of one robot and a part of a second robot, configure a `remote` on the first robot's part.
+
+1. Go to the Viam app robot page of the robot part to which you wish to establish the remote connection.
+   This is the robot part whose resources will be accessible to the other robot part.
+2. Click the **CODE SAMPLE** tab.
+3. On the **Language** toggle, select **Remotes**.
+4. Copy the `address` of the robot to your clipboard.
+
+![The Viam app CODE SAMPLE tab with Remotes selected and a copyable JSON snippet with the name, address and secret of the robot part.](../img/remotes/remote-address.png)
+
+5. Go to the Viam app robot page of the robot part from which you want to establish a remote connection.
+   This is the robot part that will be able to access the resources of the other robot part.
+6. Click the **CONFIG** tab, and then click the **REMOTES** sub-tab.
+
+![The Viam app CONFIG tab with the REMOTES sub-tab open.](../img/remotes/remote-create.png)
+
+1. Give the remote a name (you can just use the name of the other robot part, for example, "my-other-robot-main") and click **Create Remote**.
+2. Paste the `address` (for example, `my-other-robot-main.abc1de23f4.viam.cloud`) into the **Address** field.
+3. Click **Add Auth** and paste the `secret` from the other robot's **CODE SAMPLE** tab into the **Auth Key** field.
+
+![The Viam app CONFIG tab with a remote configured.](../img/remotes/remote-config.png)
+
+## Usage

```

Comment:
OK the heading level here is confusing to me.  Is this relevant to sub-parts and remote connected parts?  If so, what do we say "remote part" - can we call them that only when they are connected explicitly with a remote?  If this is supposed to be only for remotes, we need to make it more clear - "Usage" sounds generic to both.

Reply:
You're right. How about:
```suggestion
## Use a remote resource with the SDKs
```

But actually I'm not clear: for sub-parts, is there a prepending of the part name? If so, I'll need to genericize this section. @edaniels ?

Reply:
for sub parts, yes when talking to the main part

Reply:
Updated on the other PR